ocs_ci/ocs/constants.py

```diff
@@ -2696,6 +2696,8 @@
 
 # ibmcloud related constants
 IBMCLOUD_VOLUME_NAME = "ibmvolume"
+IBMCLOUD_BASE_DOMAIN_ID = "a180c88cc263d58808a739e1fdfc268a"
+IBMCLOUD_CIS_INSTANCE_NAME = "odf-qe-cis"
```

Comment:
Wouldn't be better to provide those parameters via configuration, so it will not be hardcoded just for our account? (Similarly as we have `base_domain` configuration for Azure in `azure_qe_rh_ocs_com` credentials conf.)

Reply:
I prefer to have those set as part of credential conf in ocs4-jenkins repository for our IBM Cloud account and we can then read it from config.

Reply:
This is being read in from the configuration. I've also opened an MR to add this data to the credentials